### PR TITLE
add schemaspy

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

In prep for adding SwabTypes I figure it was a good time to add schemaspy

issue https://github.com/CDCgov/prime-data-input-client/issues/224

## Changes Proposed

- add a docker container to serve schemaspy

## Screenshots / Demos
Screen Shot 2020-12-14 at 5 46 59 PM

### CODE REVIEW

**Comment by benwarfield-usds** [comment]
Huh. This is definitely not how I imagined solving this problem. Cool! 😀

File: backend/Dockerfile.schemaspy [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11 AS build
+WORKDIR /home/schemaspy
+
+RUN apt update && apt install -y graphviz
+RUN curl -H "Accept: application/zip" -L https://github.com/schemaspy/schemaspy/releases/download/v6.1.0/schemaspy-6.1.0.jar > schemaspy.jar
+RUN curl -H "Accept: application/zip" https://jdbc.postgresql.org/download/postgresql-42.2.18.jar > postgresql.jar
+
+RUN java -jar schemaspy.jar -debug -t pgsql -db simple_report -u postgres -p admin_password_for_local_dev_is_not_very_secure -host localhost:5432 -o output -s "simple_report" -dp postgresql.jar
```
Comment (benwarfield-usds) [inline backend/Dockerfile.schemaspy:8]: Maybe inject SR_DB_PORT here? Or actually, have it hit the DB over the docker network?

**Review by benwarfield-usds — APPROVED** [review]
I foresee ways in which we might want to tweak this in the future (can we get it incorporated in the wiki somehow?), but no reason not to have it available now. Woo,  schema docs! 😀

**Comment by benwarfield-usds** [comment]
For the record, community says [yes we can](https://github.community/t/how-to-updade-repo-wiki-from-github-actions/121151/2).

File: backend/Dockerfile.schemaspy (hunk 1 above)
Comment (timothybest-usds) [reply backend/Dockerfile.schemaspy]: Created a named network and set up schemaspy to access the db through it 😄

**Pull request merged**